components/body/index.styl

```diff
@@ -102,11 +101,8 @@ a.artwork
   margin-bottom 20px
   padding-top 20px
 
-section:last-of-type h1.body-section-title
-  border-bottom 1px solid gray-color
-
-section:last-of-type .body-section-content
-  border-top-style none
+section:last-of-type
```

Comment:
This is another case of a styling a generic tag name b/c this is indented at the root. In a larger app this would target all `section` tags that happen to be the last `section` in their container element—which would result in an unwanted CSS conflict. Plz use `.body-section:last-of-type` instead here.